R/zchunk_L1211.nonco2_awb_R_S_T_Y_IRR.R

```diff
@@ -1,20 +1,22 @@
 #' module_emissions_L1211.nonco2_awb_R_S_T_Y_IRR
 #'
-#' Briefly describe what this chunk does.
+#' This chunk produces irrigated and rainfed production shares, and agricultural waste burning emissions by irrigated & rainfed production shares.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L1211.nonco2_tg_R_awb_C_Y_GLU_IRR}, \code{L1211.ag_irrShare_R_C_Y_GLU_irr}. The corresponding file in the
 #' original data system was \code{L1211.nonco2_awb_R_S_T_Y_IRR.R} (emissions level1).
-#' @details Describe in detail what this chunk does.
+#' @details First, we compute the irrigated and rainfed production shares within region, GLU, and crop.
+#' Second, we sum the irrigated and rainfed production shares over the historical years to get the total by region, GLU, and crop.
+#' Third, we divide the irrigated and rainfed production shares over the historical years to get the share of irrigated and rainfed production within region, GLU, and crop.
+#' Finally, we multiply nonCO2 emissions within region, GLU, and crop by irrigated and rainfed production shares.
```

Comment:
'Finally, we multiple nonCO2 emisisons....'  is still coming up as nonCO2 and not non-CO2.